docs/bare_metal/using_bare_metal.md

```diff
@@ -11,99 +11,9 @@ Here is a code snippet that can work for both Mbed OS profiles; it prints text a
 
 <span class="notes">**Note:** To be compatible with Arm microlib, a bare metal application should not return from `main()`. In this example, the `queue.dispatch_forever()` call never returns. For more details, see [Non-returning main()](../bare-metal/using-small-c-libraries.html).</span>
 
-## 1. Creating a bare metal application
+## How to switch to baremetal
```

Comment:
How to enable baremetal or rather use ?

Reply:
`Creating an application with a baremetal profile` as the actual baremetal.md describe the same

Reply:
I reverted it to the previous name